DataFormats/CSCRecHit/test/CSCSharesInputTest.cc

```diff
@@ -67,13 +67,13 @@ void CSCSharesInputTest::analyze(const edm::Event &myEvent, const edm::EventSetu
 		perEventData[2] += track->recHitsSize();
 		perMuonData[0] += track->recHitsSize();
 		
-		for (trackingRecHit_iterator jHit = track->recHitsBegin(); jHit != track->recHitsEnd(); ++jHit) {
+        for(auto const& jHit : track->recHits()) {
```

Comment:
Indentation

Reply:
These indentations again! I realized now that neovim shows the tabbed indentation differently than GitHub, that's why I never seem to get it right...

Reply:
It seems I'll have to update this PR again anyway. Are you okay with me putting back tabbed indentation (even though I don't like it)? Then we can be sure it is consistent in our editors and GitHub.